src/ctia/task/migrate_es_stores.clj

```diff
@@ -257,6 +257,21 @@
                      (:type target-store)
                      checked-count))))))
 
+(defn check-store-index
+  [[sk sr :as store] batch-size]
+  (try
+    (log/infof "checking store: %s" sk)
+    (check-store sr batch-size)
+    (catch Exception e
+      (if-let [errors (some->> (ex-data e) :error (remove nil?))]
+        (let [message (format (str "The store %s is invalid, value cannot be coerced "
```

Comment:
I wouldn't be so definitive in that sentence. Perhaps, use "certainly a coercion issue". Because I'm not sure having a `:error` field in the ex-data would be enough to be certain this is a coercion issue.